style.css

```diff
@@ -1 +1,263 @@
+* {
+ box-sizing: border-box;
+}
+.body {
+ background-color: #e6e2e1;
+}
 
+/*Nav Bar*/
+
+.nav-flex {
+ display: flex;
+ flex-direction: row;
+ justify-content: flex-end;
+ width: 100%;
```

Comment:
Flex boxes take 100% width of their parent elements.
```suggestion
```

Reply:
Thank you so much. We are proceeding with each of these changes as suggested. This one is done.